mandatory/3-function-output.js

```diff
@@ -9,7 +9,7 @@ function getNumber() {
 
 // Add comments to explain what this function does. You're meant to use Google!
 // The concat() method is used to join two or more arrays.
-This method does not change the existing arrays, but returns a new array, containing the values of the joined arrays.
+// This method does not change the existing arrays, but returns a new array, containing the values of the joined arrays.
```

Comment:
I think the proper definition should have been , "the method is used to join two or more strings." because the test cases passes strings not arrays. I know the method also works for arrays but since also we haven't done arrays as yet....

Reply:
Thank you Jontaz77 for your comment. I'll review it and make the necessary changes.